apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss

```diff
@@ -202,3 +203,31 @@ $container-width: 309px;
     border-color: $neutral-dark !important;
   }
 }
+
+.learnMoreButton {
+  display: none !important;
+}
+
+@media screen and (max-width: $width-md) {
+  .arrowContainer {
+    display: none !important;
+  }
+
+  .centerExpandedCard {
+    display: none !important;
+  }
+
+  .learnMoreButton {
+    display: block !important;
```

Comment:
why are the `!important` tags necessary? What's overriding these styles?

Reply:
For the learn more button, there is Button styling that makes the `display: inline block` which displays the Learn More button at all times. The `!important` overrides this display.

Reply:
Sorry for dragging this out but can you add a comment explaining that? `!important` tags are a huge anti-pattern so it's good to document why they're needed when they are.